js/app.js

```diff
@@ -9,4 +9,127 @@ hamburger.addEventListener('click', () => {
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Hello-Microverse-Project',
+  },
+  {
+    id: 'work2',
+    projectName: 'Setup-Mobile-First',
+    projectInfo:
+      'Setup-Mobile-First is a project in where you built your own portfolio website using Frontend Dev Tools.',
+    ProjectIMG: [
+      'img/project_imgs/setup_portfolio.JPG',
+    ],
+    languages: ['HTML5', 'Javascript', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work3',
+    projectName: 'Pokedex',
+    projectInfo:
+      'Pokedex is a personal project that I made watching a video and practicing what I learnd in my Web Developer journey.',
+    ProjectIMG: [
+      'img/project_imgs/pokedex_preview2.gif',
+    ],
+    languages: ['Javascript', 'HTML5', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work4',
+    projectName: 'Caesar Cipher',
+    projectInfo:
+      'Caesar Cipher is a personal project that I made watching a video and practicing what I learnd in my Web Developer journey.',
+    ProjectIMG: [
+      'img/project_imgs/caesar_cipher_preview.gif',
+    ],
+    languages: ['CSS', 'HTML5', 'Github', 'Javascript'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work5',
+    projectName: 'Chronometer',
+    projectInfo:
+      'Chronometer is a personal project that I made watching a video and practicing what I learnd in my Web Developer journey.',
+    ProjectIMG: [
+      'img/project_imgs/chronometer_preview.gif',
+    ],
+    languages: ['HTML5', 'Javascript', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+  {
+    id: 'work6',
+    projectName: 'Try Boostrap',
+    projectInfo:
+      'Try Boostrap is a project in where you built your own portfolio website using Boostrap.',
+    ProjectIMG: [
+      'img/project_imgs/try_boostrap.JPG',
+    ],
+    languages: ['Boostrap', 'HTML5', 'CSS', 'Github'],
+    seeProject: 'See Project',
+    SeeLive: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+    SeeSource: 'https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first',
+  },
+];
+// Pop Up section
+const PopupWindow = document.querySelectorAll('.external_link');
+const popUp = document.querySelector('.popup');
+
+PopupWindow.forEach((p, index) => {
```

Comment:
- [ ] For the popup too, please take care of fixing the broken design, you need to stick as much as possible to the figma design, see this below
<img width="748" alt="2" src="https://user-images.githubusercontent.com/77843050/231432039-a919cb40-1d39-41d3-904d-3048840f817e.png">
- Text Color
- Alignments
- Fontweigt

On the desktop, it is more than this, see the below record
and update the style to match the given design
![6](https://user-images.githubusercontent.com/77843050/231432451-150b4ec1-bee6-4318-b962-d26dff1e2078.gif)